src/hugit/app/views.cljs

```diff
@@ -304,28 +304,45 @@
       :border {:type :line}
       :label (str " " @label " ")}
      [scrollable-list
-      {:top 0
-       :left 1
-       :right 2
-       :align :left
+      {:top         0
+       :left        1
+       :right       2
+       :align       :left
        :window-size (- rows 6)
-       :items @branches
+       :items       @branches
        :item-props-f
-       (fn [branch]
-         (if (= @current-branch branch)
-           {:style {:fg :green}}
-           {:style {:fg :white}}))
-       :selected @selected
+                    (fn [branch]
+                        (if (= @current-branch branch)
+                          {:style {:fg :green}}
+                          {:style {:fg :white}}))
+       :selected    @selected
        :custom-key-handlers
-       {["enter"] {:f (fn [idx]
-                        (let [branch (nth @branches idx)]
-                          (toast> "Checking out " branch)
-                          (evt> [:checkout-branch branch])
-                          (evt> [:router/goto :status])))
-                   :label "Checkout"
-                   :type "Action"}}
+                    {["enter"] {:f     (fn [idx]
+                                           (let [branch (nth @branches idx)]
+                                                (toast> "Checking out " branch)
+                                                (evt> [:checkout-branch branch])
+                                                (evt> [:router/goto :status])))
+                                :label "Checkout"
+                                :type  "Action"}
+                     ["n"]     {:f (fn [idx]
+                                       (evt> [:router/goto :input
+                                              {:label     "New Branch"
+                                               :on-submit (fn [branch-name]
+                                                              (print branch-name)
+                                                              (if (-> @(<sub [:repo :branches :local])
+                                                                      (set)
+                                                                      (contains? branch-name))
+                                                                ((toast> "Branch Already Exits")
```

Comment:
This doesn't seem right.To execute multiple statements one after another, use `do`:
```clojure
(do
  (expr1)
  (expr2))
```
and not
```clojure
((expr1)
 (expr2))
```